map/map.js

```diff
@@ -227,7 +228,7 @@ var props = {
 		//!steal-remove-end
 
 		// Don't send events if initalizing.
-		if (!this.__inSetup) {
+		if (!this[inSetupSymbol]) {
```

Comment:
for backwards compat should be be checking both?

Reply:
We've gone back and forth on this question. My opinion is that since this is only _reading_ the value, it should just read the Symbol. `can-construct` is still _writing_ both the Symbol and the property. That change was done 4 months ago, so by now hopefull everyone should have the latest `can-construct`. Also, Matt has changed all of the Map types to explicitly depend on the latest `can-construct`.

We will keep `can-construct` writing both until the next major release, but everything that is reading is going to be reading only the Symbol.

Reply:
map types depend on `can-construct`, but not this ...

Reply:
Not sure what you mean. This will be mixed in to some map type so it will read the Symbol from the map. If the map has the new `can-construct` it should work. Right?

Reply:
Is your concern that someone will update `can-event-queue` but not their `can-map` (or whatever map) dependency?

Reply:
Ok, going to keep both for backwards compatibility